testing_llm/CLAUDE.md

```
@@ -0,0 +1,191 @@
+# Instructions for LLM Test Execution
+
+**YOU ARE EXECUTING TEST CASES, NOT WRITING CODE**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)